Pod/SQL/SQLEntityMap.m

```diff
@@ -0,0 +1,141 @@
+//
+//  SQLEntityMap.m
+//  GoldDigger
+//
+//  Created by Felipe Lobo on 4/5/16.
+//
+
+#import <ObjectiveSugar/ObjectiveSugar.h>
+#import <GoldDigger/GDGEntity.h>
+#import "GDGRelation.h"
+#import "SQLEntityMap.h"
+#import "SQLTableSource.h"
+#import "GDGColumn.h"
+#import "GDGBelongsToRelation.h"
+#import "GDGHasManyRelation.h"
+#import "SQLQuery.h"
+#import "GDGHasOneRelation.h"
+#import "GDGEntity+SQL.h"
+#import "SQLEntityQuery.h"
+
+@implementation SQLEntityMap
+
+@synthesize entityClass = _entityClass;
+
++ (instancetype)mapWithDictionary:(NSDictionary *)fromToDictionary from:(id <SQLSource>)source to:(Class)class
+{
+	SQLTableSource *table = (SQLTableSource *) source;
```

Comment:
Why not use the correct type for the `source` param and remove that hard/unsafe cast?